src/main/java/ru/sbt/mipt/oop/main/AlarmDecorator.java

```diff
@@ -0,0 +1,25 @@
+package ru.sbt.mipt.oop.main;
+
+import static ru.sbt.mipt.oop.main.SensorEventType.ALARM_ACTIVATE;
+import static ru.sbt.mipt.oop.main.SensorEventType.ALARM_DEACTIVATE;
+
+public class AlarmDecorator implements EventManagable {
+
+    private EventManagable wrappee;
+
+    public AlarmDecorator(EventManagable wrappee) {
+        this.wrappee = wrappee;
+    }
+
+    @Override
+    public void manage(SensorEvent event, SmartHome smartHome) {
+        if (((event.getType() != ALARM_ACTIVATE) || event.getType() != ALARM_DEACTIVATE)
```

Comment:
https://github.com/iagoshkov/smart-home-2020/blob/master/STUDENT_GUIDE.md#%D1%81%D0%B4%D0%B5%D0%BB%D0%B0%D0%B9%D1%82%D0%B5-%D0%BE%D1%82%D0%BF%D1%80%D0%B0%D0%B2%D0%BA%D1%83-%D0%BC%D0%B0%D0%BA%D1%81%D0%B8%D0%BC%D1%83%D0%BC-%D0%BE%D0%B4%D0%BD%D0%BE%D0%B3%D0%BE-%D1%81%D0%BC%D1%81-%D0%B8%D0%B7-%D0%B4%D0%B5%D0%BA%D0%BE%D1%80%D0%B0%D1%82%D0%BE%D1%80%D0%B0